### PR TITLE
[WebProfilerBundle] Expose dotenv variables

### PR DESCRIPTION
**Pull request by ro0NL**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes/no
| Fixed tickets | #23951
| License       | MIT
| Doc PR        | symfony/symfony-docs#... 

Variant of #23951

image

### CODE REVIEW

**Comment by sroze** [comment]
I like that idea 👍

**Comment by nicolas-grekas** [comment]
3.4 is bugfix only now

**Review by nicolas-grekas — APPROVED** [review]

**Comment by sroze** [comment]
@nicolas-grekas why did you approve it then? 😛

**Comment by nicolas-grekas** [comment]
To let others resolve this :)

**Comment by xabbuh** [comment]
@ro0NL Looks like we need a rebase now. :)

**Comment by fabpot** [comment]
@ro0NL Can you rebase on current master?

**Comment by ro0NL** [comment]
Done. Perhaps both h4 headings added need some thought for better naming.

File: src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/request.html.twig [hunk 1]
```diff
@@ -167,7 +167,15 @@
                 {% endif %}
 
                 <h3>Server Parameters</h3>
-                {{ include('@WebProfiler/Profiler/bag.html.twig', { bag: collector.requestserver }, with_context = false) }}
+                <h4>Managed by Symfony Dotenv</h4>
```
Comment (fabpot) [inline src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/request.html.twig]: `Defined in .env`?

File: src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/request.html.twig [hunk 2]
```diff
@@ -167,7 +167,15 @@
...
+                {{ include('@WebProfiler/Profiler/bag.html.twig', { bag: collector.dotenvvars }, with_context = false) }}
+
+                <h4>Standard variables</h4>
```
Comment (fabpot) [inline src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/request.html.twig]: `Defined as regular env variables`?

File: src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/request.html.twig (hunk 1 above)
Comment (ro0NL) [reply src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/request.html.twig]: Lets go from there yes. Now updated.

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @ro0NL.

**Pull request closed without merging**